### PR TITLE
add-design/feat/footer-feat/routing

### PR DESCRIPTION
**Pull request by AlinaHauriluk**
added footer and routing

### CODE REVIEW

File: client/src/components/footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  FooterContactInfo,
+} from "./footer";
+import { FlexContainerSpace, FlexStartColumn } from "../../box/flexBox";
+
+const Footer = () => {
+  const contacts = [
+    { id: 1, title: "Address:", info: "481 Creekside Lane Avila CA" },
+    { id: 2, title: "Phone:", info: "+53 345 7953 32453" },
+    { id: 3, title: "Email:", info: "yourmail@gmail" },
+  ];
```
Comment (tatsianamatesha) [inline client/src/components/footer/index.tsx]: you can move this constant above component initialisation. we don't need to initialise this constant on each render.

File: client/src/pages/about/index.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import DefaultLayout from "../../layout/defaultLayout";
+
+const About = () => {
+  return(
```
Comment (tatsianamatesha) [inline client/src/pages/about/index.tsx]: you can write 
``` 
const About = () => (
  <DefaultLayout>
  ...
  </DefaultLayout>
 ```

File: client/src/pages/cart/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import DefaultLayout from "../../layout/defaultLayout";
+
+const Cart = () => {
+  return(
+    <DefaultLayout>
```
Comment (tatsianamatesha) [inline client/src/pages/cart/index.tsx]: the same is here

File: client/src/pages/contacts/index.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import DefaultLayout from "../../layout/defaultLayout";
+
+const Contacts = () => {
+  return(
```
Comment (tatsianamatesha) [inline client/src/pages/contacts/index.tsx]: the same is here

File: client/src/pages/menu/index.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import DefaultLayout from "../../layout/defaultLayout";
+
+const Menu = () => {
+  return(
```
Comment (tatsianamatesha) [inline client/src/pages/menu/index.tsx]: the same is here

File: client/src/pages/profile/index.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import DefaultLayout from "../../layout/defaultLayout";
+
+const Profile = () => {
+  return(
```
Comment (tatsianamatesha) [inline client/src/pages/profile/index.tsx]: the same is here

File: client/src/components/footer/index.tsx (hunk 1 above)
Comment (AlinaHauriluk) [reply client/src/components/footer/index.tsx]: agree with you

File: client/src/pages/about/index.tsx (hunk 2 above)
Comment (AlinaHauriluk) [reply client/src/pages/about/index.tsx]: i understood it is more rationally

**Review by tatsianamatesha — APPROVED** [review]

**Pull request merged**